cms/templates/export.html

```diff
@@ -6,18 +6,25 @@
   from django.utils.translation import ugettext as _
   import json
 %>
-<%block name="title">${_("Course Export")}</%block>
+<%block name="title">
+%if library:
```

Comment:
Ah ok so these aren't particularly places that need ngettext. I think honestly, though, for translation purposes it'd be better to just have the strings as they are. Adding a lot of variables that you need to contextually understand that are themselves translated seems like a much larger overhead, and may result in poorly translated strings. My instinct is to not do as you suggest.

As far as capitalization rules, I would follow whatever the doc team says. Generally I would expect titles to have capitalization (eg, "Library Export") but in-line in a sentence, seems appropriate to use lowercase.